Make gem binaries on ruby 1.9 use a 19 suffix, so you can have both
1.8 and 1.9 versions of the same gem installed at the same time
without conflicts.

--- lib/rubygems/commands/install_command.rb.orig	Tue Jul 26 19:04:03 2011
+++ lib/rubygems/commands/install_command.rb	Sun Oct 30 19:10:20 2011
@@ -21,7 +21,7 @@ class Gem::Commands::InstallCommand < Gem::Command
     defaults = Gem::DependencyInstaller::DEFAULT_OPTIONS.merge({
       :generate_rdoc     => true,
       :generate_ri       => true,
-      :format_executable => false,
+      :format_executable => true,
       :version           => Gem::Requirement.default,
     })
 
